### PR TITLE
Improve NewRelic transaction data

### PR DESCRIPTION
**Pull request by thelollies**

### CODE REVIEW

File: lib/hoodoo/services/middleware/middleware.rb [hunk 1]
```diff
@@ -413,7 +413,6 @@ def initialize( app )
          service_container.is_a?( NewRelic::Agent::Instrumentation::MiddlewareProxy )
 
         if service_container.respond_to?( :target )
-          newrelic_wrapper  = service_container
           service_container = service_container.target()
```
Comment (pond) [inline lib/hoodoo/services/middleware/middleware.rb]: Good catch, left-over from a subsequent refactor ages ago.

**Comment by pond** [comment]
This looks good and I'm happy to merge as-is if you think it's ready to go.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 1.8.3 (2016-05-06)
+
+* NewRelic traces now include the action and path to enable better filtering.
+* The dispatch method in Hoodoo::Services::Middleware is marked for tracing so that the time spent in the implementation can be distinguished from the time spent in the middleware.
```
Comment (pond) [inline CHANGELOG.md]: No it isn't; the log-inbound method is :-) perhaps give new & old method names in `backticks` for clarity

File: lib/hoodoo/monkey/patch/newrelic_middleware_analytics.rb [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+#           visibility over the time spent in implementations separately
+#           from the time spent in middleware.
+#
+#           See Hoodoo::Monkey::Patch::NewRelicMiddlewareAnalytics for
+#           more.
+# ----------------------------------------------------------------------
+#           06-May-2016 (RJS): Created.
+########################################################################
+
+
```
Comment (pond) [inline lib/hoodoo/monkey/patch/newrelic_middleware_analytics.rb]: (Super-trivial) delete spurious additional white space vertical - one line gap only

File: CHANGELOG.md (hunk 2 above)
Comment (thelollies) [inline CHANGELOG.md]: monkey_log_inbound_request has custom attributes. Dispatch is marked for tracing.

**Pull request merged**